sys/linux/auto.txt

```diff
@@ -2127,9 +2125,9 @@ file_handle$auto_record {
 
 flow_policy$auto_datapath [
 	OVS_FLOW_ATTR_KEY		nlnest[OVS_FLOW_ATTR_KEY, array[nl_generic_attr]]
+	OVS_FLOW_ATTR_MASK		nlnest[OVS_FLOW_ATTR_MASK, array[nl_generic_attr]]
```

Comment:
The current order is based on the source code (should match how they are described in the policy). There may be some logical grouping, also easier to match with source code if you are looking at both.
So I don't see alphabetical order as superior.
We do use sorting to prevent non-determinism, but it's not the case here.